software_project_management/fair/01_understanding_fair.md

```diff
@@ -0,0 +1,399 @@
+dependsOn: []
+tags: []
+learningOutcomes:
+  - Understand the 4 foundational principles of FAIR
+  - Be able to identify actionable items to improve the FAIRness of a project
+attribution:
+  - citation: The FAIR Guiding Principles for scientific data management and
+    stewardship
+    url: https://doi.org/10.1038/sdata.2016.18
+    image: https://media.springernature.com/full/nature-cms/uploads/product/sdata/header-87021870c315c48063927b82055c12bc.svg
+    license: CC-BY-4.0
+  - citation: GO-FAIR
+    url: https://www.go-fair.org/fair-principles/
+    image: https://www.go-fair.org/wp-content/themes/go-fair/images/logo.svg
+    license: CC-BY-4.0
+  - citation: FAIRsharing
+    url: https://fairsharing.org
+    image: https://fairsharing.org/assets/fairsharing-logo.svg
+    license: CC-BY-4.0
+  - citation: OpenAIRE
+    url: https://www.openaire.eu/how-to-make-your-data-fair
+    image: https://www.openaire.eu/templates/yootheme/cache/19/Logo_Horizontal-1910c000.webp
+    license: CC-BY-4.0
+  - citation: FAIR toolkit
+    url: https://fairtoolkit.pistoiaalliance.org
+    image: https://fairtoolkit.pistoiaalliance.org/wp-content/uploads/2020/05/FAIR-Toolkit-logo.png
+    license: CC-BY-4.0
+  - citation: How to FAIR a Danish website to guide researchers on making
+    research data more FAIR
+    url: https://doi.org/10.5281/zenodo.3712065
+    image: https://www.howtofair.dk/media/e5md4otg/htfair_logo.svg
+    license: CC-BY-4.0
+---
+## Introduction
+
+This module aims to give you an overview of the FAIR principles for scientific
+data. We will start with some scenarios that researchers frequently face, the
+motivation of the FAIR principles and their implication to your research. By
+the end of this module, you will be able to explain the FAIR principles in your
+own words and identify any existing FAIRness gaps in your data with some
+recommendations to improve it.
+
+## Common Scenarios
+
+As a researcher, the following scenarios may be familiar to you:
+
+- You want to replicate some analyses by other researchers but either the data
+  is lost or incomplete.
+- You are not sure if you can reuse or how to access those dataset even they
+  are available.
+- There is no documentation or metadata associated with the dataset which you
+  are going to reuse.
+- You do not know how other researchers obtained or generated the data
+  (provenance)
+- You try to explore dataset similar to your research but you have no clue
+  about how to locate relevant dataset.
+- You have your paper published but not sure where or how to put the associated
+  data for others to verify or reuse.
+
+The FAIR principles data aim to alleviate some of the above pains arising from
+the above scenarios.
+
+## FAIR Guiding Principles
+
+FAIR is composed of 4 foundational principles:
+
+1. To be Findable
+2. To be Accessible
+3. To be Interoperable
+4. To be Reusable
+
+Modern scientific research often involve handling data that far exceed the
```

Comment:
Minor typo, involve should be plural, i.e. 'Modern scientific research often involve**s** handling data'